hacks/setup-env.sh

```diff
@@ -0,0 +1,7 @@
+REACT_APP_API_URL="http://REACT_APP_API_URL.env.variable.not:set"
+REACT_APP_VERSION="$(node -e 'console.log(require("./node_modules/openshift-assisted-ui-lib/package.json").version)')"
```

Comment:
Maybe this a faster alternative?

`cat node_modules/openshift-assisted-ui-lib/package.json | jq -r '.version'`

Reply:
Yes, but that assumes that you are using Linux and have `jq` installed.
No matter what OS you will use in order to execute the project, you must have node installed there.

Reply:
True also, specially `jq` would not be available in most cases.